src/modules/permissions/infrastructure/repositories/permission.vm.repository.ts

```
@@ -96,4 +96,14 @@ export class PermissionVmRepository
     await this.findPermissionByIds(vmId, roleId);
     await this.delete({ vmId, roleId });
   }
+
+  async deleteById(id: string): Promise<void> {
+    const result = await this.delete({ id });
+    console.log(`Deleted permission with id ${id}, result:`, result);
+  }
+
+  async deleteByRoleId(roleId: string): Promise<void> {
+    const result = await this.delete({ roleId });
+    console.log(`Deleted all VM permissions for role ${roleId}, result:`, result);
```

ESLint

[error] Replace ``Deleted·all·VM·permissions·for·role·${roleId},·result:`,·result` with `⏎······`Deleted·all·VM·permissions·for·role·${roleId},·result:`,⏎······result,⏎····`

(prettier/prettier)